ex01.exs

```diff
@@ -28,10 +28,27 @@ defmodule Ex01 do
   """
   
   def counter(value \\ 0) do
+    receive do
+      { :next, self } -> send self, { :next_is, value }
+    end
+    counter value + 1
```

Comment:
No deduction, but I personally prefer to see parentheses in this case:

~~~ elixir
counter(value + 1)
~~~

To my eye it seems to make easier to see what's going on.